app/screens/post_priority_picker/footer.tsx

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {BottomSheetFooter, BottomSheetFooterProps} from '@gorhom/bottom-sheet';
+import React from 'react';
+import {Platform, TouchableOpacity, View} from 'react-native';
+
+import FormattedText from '@components/formatted_text';
+import {useTheme} from '@context/theme';
+import {useIsTablet} from '@hooks/device';
+import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
+import {typography} from '@utils/typography';
+
+export type Props = BottomSheetFooterProps & {
+    onCancel: () => void;
+    onSubmit: () => void;
+}
+
+const getStyleSheet = makeStyleSheetFromTheme((theme: Theme, isTablet: boolean) => ({
```

Comment:
This is not going to work, specially for foldable devices.

The way makeStyleSheetFromTheme works, it just regenerate the style if the theme changes. The rest of the arguments are not taken into consideration. So if the value of `isTablet` changes during the execution (foldable devices and some windowed modes) the styles won't change.

What we do in other parts of the code is handle tablet differences in the body of the component. Another option would be to adapt the `makeStyleSheetFromTheme` to accept a dependencies list. Or even transform it to a hook. But those options probably are out of the scope of this PR.

Reply:
Yep I found out it couldn't work when I read the function better 😓 thanks.
The dependency array is a good idea. Still I'm going by inlining those styles